### PR TITLE
bump patch

### PR DESCRIPTION
**Pull request by FynnBe**
patch bump needed for bioimageio.core as post seems to be ignored in version resolution: https://github.com/bioimage-io/core-bioimage-io-python/pull/140/files#r744832636

### CODE REVIEW

**Comment by constantinpape** [comment]
This should not be necessary, see my comments in https://github.com/bioimage-io/core-bioimage-io-python/pull/140/files#r744832636

**Comment by constantinpape** [comment]
@FynnBe I guess we can revive this and bump the patch due to the addition of `eps`. (It's a tiny change, but I think we should stick to our convention here.)

File: README.md [hunk 1]
```diff
@@ -140,6 +140,9 @@ or if the field `test_inputs` does not contain a list, it would print:
 
 
 ## Changelog
+ * **0.3.4**:
+   - Other changes
+      - Add parameter `eps` to `scale_range` postprocessing.
```
Comment (FynnBe) [inline README.md:145]:
```suggestion
      - Add optional parameter `eps` to `scale_range` postprocessing.
```

**Review by FynnBe — COMMENTED** [review]
maybe add "optional". GH trying to tell me I can't approve (my own PR), but here I go: APPROVED 👍

**Pull request merged**